MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -719,7 +719,7 @@ internal void CreateSizeDependentResources(bool useFullscreenParameter = false)
                 using (var dxgiFactory = dxgiAdapter.GetParent<SharpDX.DXGI.Factory1>())
                 {
                     _swapChain = new SwapChain(dxgiFactory, dxgiDevice, desc);
-
+                    dxgiFactory.MakeWindowAssociation(PresentationParameters.DeviceWindowHandle, WindowAssociationFlags.IgnoreAll);
```

Comment:
I would agree with that.  I was wondering where the `Alt+Enter` key combo was being handled.